packages/client/src/fm-player/services/connected-users-service.ts

```diff
@@ -0,0 +1,24 @@
+import mitt, { Emitter, Handler } from 'mitt';
+
+const emitter: Emitter = mitt();
+
+let connectedUsers: string[] = [];
+
+function getUsers(): string[] {
+  return [...connectedUsers];
```

Comment:
We actually could do some TypeScript magic here:
```suggestion
  return [...connectedUsers] as const;
```

It marks the array as mutable, this way type checker will scream if we every try to change it.

It also changes the return type of that function to `readonly string[]`.